apps/teams-test-app/index_local.html

```diff
@@ -0,0 +1,33 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Teams Test App created using React"
+    />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="./public/manifest.json" />
+    <title>Teams Test App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <script src="MicrosoftTeams.min.js"></script>
```

Comment:
@ramourt Another place where we can make an improvement (depending on how this file is used) to use the integrity hash that is auto-generated in Murthy's PR and then also update it automatically as part of generating it.